coa-helm/templates/tes.yaml

```diff
@@ -17,6 +17,8 @@ spec:
       labels:
         io.kompose.service: tes
         aadpodidbinding: {{ .Values.identity.name }}
+        leoServiceName: tes
+    {{ include "coa-helm.labels" $ | nindent 8 }}
```

Comment:
TES didn't have the standard labels including `leoAppName`, `workspaceId` -- so added that as well.

Reply:
This indent looks different than in the other files but `nindent` is still 8. Is that right?